syntax = "proto3";

message UserDeleted {
    string user_id = 1;
}